pegasus/sites.v3/csedweek.org/public/about/index.md

```diff
@@ -1,63 +0,0 @@
----
```

Comment:
what indication do we have that csedweek.org is no longer being used? I could be missing something but it looks to me like we might not be tracking this entire domain name with GA.

Reply:
I talked with Brendan, and it sounds like although csedweek.org is still a live web site, it is no longer hosted by us. so, that confirms that this directory is safe to delete.

based on that, my request would be to split the removal of the csedweek.org directory off into a separate PR, since this PR is getting quite big, and keeping it all together will add to the difficulty if we break something and have to undo some of this work. sorry for the added work, but I do think it'll give us a bit more confidence in shipping all of this.

Reply:
This works for me! I'll reinstate this directory and make a new PR.